### PR TITLE
[FAB-17265] Move private data tutorial to use test network

### PR DESCRIPTION
**Pull request by nikhil550**
Signed-off-by: NIKHIL E GUPTA <negupta@us.ibm.com>

#### Type of change

- Documentation update

#### Description

Move the private data tutorial from the first-network sample to the test network as part of the long term move away from BYFN

### CODE REVIEW

File: docs/source/private_data_tutorial.rst [hunk 1]
```diff
@@ -117,7 +110,7 @@ The data to be secured by these policies is mapped in chaincode and will be
 shown later in the tutorial.
 
 This collection definition file is deployed when the chaincode definition is
-committed to the channel using the `peer lifecycle chaincode commit command <http://hyperledger-fabric.readthedocs.io/en/latest/commands/peerchaincode.html#peer-chaincode-instantiate>`__.
+committed to the channel using the `peer lifecycle chaincode commit command <file:///Users/nikhilgupta/fabric/docs/build/html/commands/peerlifecycle.html#peer-lifecycle-chaincode-commit>`__.
```
Comment (mastersingh24) [inline docs/source/private_data_tutorial.rst]: Looks like you are referencing a file on your system rather than the web link?

Comment (mastersingh24) [inline docs/source/private_data_tutorial.rst]:
```suggestion
committed to the channel using the `peer lifecycle chaincode commit command <http://hyperledger-fabric.readthedocs.io/en/latest/commands/peerlifecycle.html#peer-lifecycle-chaincode-commit>`__.
```

**Review by mastersingh24 — CHANGES_REQUESTED** [review]

File: docs/source/private_data_tutorial.rst (hunk 1 above)
Comment (nikhil550) [reply docs/source/private_data_tutorial.rst]: 😬 Looks like I got the link from the wrong browser window. Thanks for the catch.

File: docs/source/private_data_tutorial.rst [hunk 2]
```diff
@@ -256,16 +249,26 @@ private data.
 :guilabel:`Try it yourself`
 
 Before installing, defining, and using the marbles private data chaincode below,
-we need to start the BYFN network. For the sake of this tutorial, we want to
-operate from a known initial state. The following command will kill any active
+we need to start the Fabric test network. For the sake of this tutorial, we want
+to operate from a known initial state. The following command will kill any active
 or stale docker containers and remove previously generated artifacts.
```
Comment (mastersingh24) [inline docs/source/private_data_tutorial.rst]:
```suggestion
or stale Docker containers and remove previously generated artifacts.
```

File: docs/source/private_data_tutorial.rst [hunk 3]
```diff
@@ -277,13 +280,14 @@ the following commands to clean up previous environments:
...
+up the Fabric test network with CouchDB:
 
 .. code:: bash
 
-   ./byfn.sh up -c mychannel -s couchdb
+   ./network.sh up createChannel -s couchdb
 
-This will create a simple Fabric network consisting of a single channel named
+This command will deploy a Fabric network consisting of a single channel named
 ``mychannel`` with two organizations (each maintaining two peer nodes) and an
```
Comment (mastersingh24) [inline docs/source/private_data_tutorial.rst]:
```suggestion
``mychannel`` with two organizations (each maintaining one peer node) and an
```
I think there's only one peer per org?

File: docs/source/private_data_tutorial.rst [hunk 4]
```diff
@@ -367,37 +365,20 @@ the response below:
...
-
-.. code:: bash
-
-    export CORE_PEER_LOCALMSPID=Org2MSP
-    export PEER0_ORG2_CA=/opt/gopath/src/github.com/hyperledger/fabric/peer/crypto/peerOrganizations/org2.example.com/peers/peer0.org2.example.com/tls/ca.crt
-    export CORE_PEER_TLS_ROOTCERT_FILE=$PEER0_ORG2_CA
-    export CORE_PEER_MSPCONFIGPATH=/opt/gopath/src/github.com/hyperledger/fabric/peer/crypto/peerOrganizations/org2.example.com/users/Admin@org2.example.com/msp
-
-5. Switch the active peer to the first peer in Org2 and install the chaincode:
+3. Now use the CLI operate as the Org2 admin. Copy and paste the following block of
```
Comment (mastersingh24) [inline docs/source/private_data_tutorial.rst]:
```suggestion
3. Now use the CLI operating as the Org2 admin. Copy and paste the following block of
```

or

```suggestion
3. Now use the CLI as the Org2 admin. Copy and paste the following block of
```

File: docs/source/private_data_tutorial.rst [hunk 5]
```diff
@@ -367,37 +365,20 @@ the response below:
...
 
-    export CORE_PEER_ADDRESS=peer0.org2.example.com:9051
-    peer lifecycle chaincode install marblesp.tar.gz
+    export CORE_PEER_LOCALMSPID="Org2MSP"
+    export CORE_PEER_TLS_ROOTCERT_FILE=${PWD}/organizations/peerOrganizations/org2.example.com/peers/peer0.org2.example.com/tls/ca.crt
+    export CORE_PEER_MSPCONFIGPATH=${PWD}/organizations/peerOrganizations/org2.example.com/users/Admin@org2.example.com/msp
+    export CORE_PEER_ADDRESS=localhost:9051
 
-6. Switch the active peer to the second peer in org2 and install the chaincode:
+4. Run the following command install the chaincode on the Org2 peer:
```
Comment (mastersingh24) [inline docs/source/private_data_tutorial.rst]:
```suggestion
4. Run the following command to install the chaincode on the Org2 peer:
```

**Review by mastersingh24 — DISMISSED** [review]

File: docs/source/private_data_tutorial.rst (hunk 2 above)
Comment (nikhil550) [reply docs/source/private_data_tutorial.rst]: Done, found somewhere else it was lowercase...

File: docs/source/private_data_tutorial.rst (hunk 3 above)
Comment (nikhil550) [reply docs/source/private_data_tutorial.rst]: yes. done

File: docs/source/private_data_tutorial.rst (hunk 4 above)
Comment (nikhil550) [reply docs/source/private_data_tutorial.rst]: done.

File: docs/source/private_data_tutorial.rst (hunk 5 above)
Comment (nikhil550) [reply docs/source/private_data_tutorial.rst]: done.

**Review by pamandrejko — COMMENTED** [review]
I ran through all the steps of the tutorial and everything worked except for when I tried to bring up the test network using `./network.sh up createChannel -s couchdb` because I had some VSCode peers running which interfered with Nik's test network. 

He is making a fix to test network so users would not experience the problem. 

Otherwise, everything worked perfectly.

Thanks Nik for doing this.

**Comment by nikhil550** [comment]
Updated images as well.

File: docs/source/private_data_tutorial.rst [hunk 6]
```diff
@@ -795,36 +750,23 @@ price private data is purged.
 
 :guilabel:`Try it yourself`
 
-Switch back to peer0 in Org1 using the following commands. Copy and paste the
-following code block and run it inside your peer container:
+Switch back Org1 using the following commands. Copy and paste the following code
```
[the remark this answers is not in the document]
Comment (nikhil550) [reply docs/source/private_data_tutorial.rst]: done.

**Review by denyeart — APPROVED** [review]

**Pull request merged**